### DISCUSSION TITLE
Statamic Admin Form for Entering Data into Eloquent Models

### DISCUSSION THREAD

**Discussion by circleb** [statamic/cms#4663] [Q&A]
I've got a fairly complex Laravel project with lots of eloquent models and relations, API endpoints, etc...
I'd really like to use the control panel and fields that come with statamic to enter data into the tables for those models.
I'm probably not searching for the right terminology, because I've not been able to find any examples for that use case.

**Comment by circleb** [comment]
I figured I'd give a little more background so this isn't an x/y request!
What I have is a large amount of data from an old system, that I've brought into Laravel for the main purpose of being accessible through an API. I'd intended to build my own admin pages to add/update the data, but came across statamic and really like how much of the control panel is ready to go. I'm thinking it would be really nice to tie it into my existing data.

**Comment by duncanmcclean** [comment]
Not sure I 100% understand the request but if you're looking to manage those models inside the Control Panel, you might be able to use Runway. https://statamic.com/addons/double-three-digital/runway

_(Disclaimer: I'm the developer of Runway)_

**Comment by jackmcdade** [comment]
Are you looking to USE the control panel, or just the components FROM the control panel?
